domain/src/main/java/org/mobilitydata/gtfsvalidator/domain/entity/RouteType.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2020. MobilityData IO.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.mobilitydata.gtfsvalidator.domain.entity;
+
+import java.util.stream.Stream;
+
+public enum RouteType {
+    LIGHT_RAIL(0),
+    SUBWAY(1),
+    RAIL(2),
+    BUS(3),
+    FERRY(4),
+    CABLE_CAR(5),
+    GONDOLA(6),
+    FUNICULAR(7);
+
+    private int value;
+
+    RouteType(int value) {
+        this.value = value;
+    }
+
+    // no implementation for unsupported enum value
+
+    static public RouteType fromInt(Integer fromValue) {
```

Comment:
This method (and all similar enum methods for GTFS entities) should be benchmarked for performance at some point, as it will be invoked for every line in the GTFS file - see https://github.com/MobilityData/gtfs-validator/issues/48.

Reply:
#48